package.json

```diff
@@ -11,6 +11,7 @@
     "cypress-run": "npx cypress run"
   },
   "dependencies": {
+    "@lottiefiles/lottie-player": "^2.0.4",
```

Comment:
theres a weird merge error here, suggest merging with dev now that firebase is added